ideascube/templates/ideascube/includes/tag_cloud.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n ideascube_tags %}
 {% spaceless %}
 <div class="card tinted">
-    <h4><span class="theme learn">{% trans "browse" %}</span> {% trans "Topics" %}</h4>
+    <h4><span class="theme discover">{% trans "browse" %}</span> {% trans "Topics" %}</h4>
```

Comment:
I don't understand why this commit is necessary. Can you provide a before/after screenshot? Or maybe explain where in the application are those "other titles" displayed, so I can go and look at the difference for myself?